tools/virt-builder/about.md

```diff
@@ -47,38 +47,45 @@ root password, only the bare minimum installed software, etc.).
 
 Import the above disk image into libvirt:
 
-    $ virt-install --import \
-       --name f23vm1 --ram 2048 \
-       --disk path=fedora-23.img,format=raw --os-variant fedora23
+    $ virt-install \
+       --name f27vm1 --ram 2048 \
+       --disk path=fedora-27.img,format=raw \
+       --os-variant fedora26 \
+       --import \
```

Comment:
Please insert this into triple backticks. For the code to look
```
like this.
```

Please do also leave out the final `\`, as it may confuse some users when pasting the code.

Reply:
Oh, you're right. Will do.